composer.json

```diff
@@ -76,6 +76,7 @@
         },
         "files": [
             "src/functions.php",
+            "src/command_functions.php",
```

Comment:
The reason behind this is that the newly added functions are used both in binaries ( before including autoload.php ) and within internal classes.

without this, psalm would work as excepted, however, if someone is working on a project that extends psalm itself by using it as a non-dev dependency and using `Psalm\Config`, they would need the newly added functions as well.

Reply:
Hm, not sure what the solution is here, currently breaking as composer requires this using `require` instead of `require_once`

Reply:
@weirdan do you think redeclaring the functions twice is the way to go here? I can't come up with another solution ...

Reply:
I would probably suggest to add those functions to an autoloadable class (as a static method), and `require_once` that file in binaries (or `command_functions`).

Reply:
I have added an internal class ( `Psalm\Internal\Compposer` ), which we could use later to add more functions that can take care of tracking down `vendor` and `autoload.php`.